openstack/cronus/templates/cronus/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.cronus.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Didn't we agree that we'll use statefulset?

Reply:
{{- if .Values.cronus.enabled }}
apiVersion: apps/v1
{{ if .Values.cronus.statefulSet.enabled }}
kind: StatefulSet
{{ else }}
kind: Deployment
{{ end -}}